playbooks/generic-patch_upstream.yml

```diff
@@ -3,16 +3,20 @@
   gather_facts: yes
   any_errors_fatal: true
   pre_tasks:
-    #http://jinja.pocoo.org/docs/2.10/templates/#default
-    - name: Check if developer mode should be enabled
+    - name: Check if upstream patches should be applied
```

Comment:
Should still be "checking if developer mode is set"?

Reply:
Good point, fixed